### ISSUE TITLE
Nemo sidebar in places view not working correclty when clicking

### ISSUE THREAD

**Issue by zboraon** [linuxmint/nemo#1968]
```
 * Nemo version (nemo --3.8.6)
 * Is issue with desktop or windowed nemo? windowed nemo
 * Distribution - (Mint 19)
 * Graphics hardware *and* driver used Intel cpu
 * 64 bit
 ```

**Issue**
In nemo, through the sidebar view option as places, when I click to bottom folders (for example to bookmarks in not maximized screen), sidebar immediately takes its position as the default position
![47147879-dcdbe400-d2d7-11e8-8f3f-b06185b30163](https://user-images.githubusercontent.com/22004884/47716554-16a3d780-dc54-11e8-8e1e-f7433c507f5e.png)

**Steps to reproduce**
In nemo places view add bookmarks and extend the list and then click to one of them at the bottom when notin  maximized

**Expected behaviour**
It shouldn't jump to top

**Other information**

**Comment by mtwebster** [comment]
I added a fix for this issue.  It's not ideal, but it's the best I can do for now until our next dev cycle, when I can fix it properly (so the sidebar does not move when you make a selection).

I'll leave this open for now as a reminder.

**Comment by xenopeek** [comment]
@mtwebster I think this can be closed?

Testing with Nemo 4.6.4 I see that clicking a bookmark doesn't cause the sidebar to jump to the top. The sidebar does jump a bit to place the clicked bookmark about center vertically in the sidebar.

**Comment by mtwebster** [comment]
I did a little bit better fix: 557b7bbfdf69 and reverted 3750bfffe6e.  The sidebar shouldn't jump at all except for mount/unmount events (which is a bit unavoidable).
